MICP_Assignments/Assignment_Week_4.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+

```

Comment:
Thanks for uploading your code/test cases! Don't forget to upload your pen and paper TEBOW IT for future assignments (or include pictures of your TEBOW IT on a whiteboard)! Writing the problem out by hand and timing yourself to 45 minutes is one of the best ways to practice for a real interview.